packages/mdctl-axon-tools/lib/StudyDataTranslations.js

```diff
@@ -34,6 +34,10 @@ class StudyDataTranslations {
       }
     }
 
+    if (!manifestData) {
+      throw Fault.create('mdctl.kManifestNotFound.error', { reason: 'There is no manifest set as parameter neither found in directory' })
```

Comment:
The error message seems ambiguous to me, what about this?
`No manifest found in the parameters and in the import directory`

Reply:
Sounds the same to me

Reply:
I guess the grammatically correct version should be  "There is no manifest set as a parameter, nor is one found in the directory."

Reply:
@imaimai86 @joejean just updated the error message as u guys suggested.